recipes/pdspy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+    name: pdspy
+    version: "1.5.0"
+
+source:
+    url: https://files.pythonhosted.org/packages/ee/b0/3e834fb25a597ef12a9764b92ee1c5cf6e1d4897805c9c1346ac0f70dfbf/pdspy-1.5.0.tar.gz
+    sha256: 94d68282132f0272ef41fa4da84eebbf859ead2f3cb407223feeb9f2d1a7a247
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+
+requirements:
```

Comment:
you need a `build` section like the following:

```
requirements:
  build:
    - {{ compiler('c') }}
    - {{ compiler('fortran') }}
```

assuming you need both a c and fortran compiler. If you don't need the C-compiler you can drop the corresponding line.

Reply:
I've added both as both should be needed.